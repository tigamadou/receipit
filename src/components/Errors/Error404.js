import React from 'react';

const Error404 = () => (
  <div className="container">404</div>
);

export default Error404;
