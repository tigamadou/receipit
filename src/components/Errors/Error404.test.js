import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';

import Error404 from './Error404';

describe('Error404 component tests', () => {
  test('renders Error404 component ', () => {
    render(<Error404 />);
    expect(screen.getByText('404')).toBeInTheDocument();
  });

  test('should match with snapshot', () => {
    const tree = renderer
      .create(<Error404 />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
