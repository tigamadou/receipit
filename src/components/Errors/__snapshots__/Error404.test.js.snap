// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error404 component tests should match with snapshot 1`] = `
<div
  className="container"
>
  404
</div>
`;
